projects/cpython3/project.yaml

```diff
@@ -2,4 +2,8 @@ homepage: "https://python.org/"
 primary_contact: "gps@google.com"
 auto_ccs:
```

Comment:
Do you want to add yourself here?

Reply:
Gotta check with the python security team, I'll add myself after they give the go-ahead.